src/libexpr/flake/flake.cc

```diff
@@ -599,4 +635,30 @@ Fingerprint LockedFlake::getFingerprint() const
 
 Flake::~Flake() { }
 
+void ConfigFile::apply()
+{
+    for (auto & [name, value] : options) {
+        // FIXME: support 'trusted-public-keys' (and other options), but make it TOFU.
+        if (name != "bash-prompt-suffix" &&
+            name != "bash-prompt" &&
+            name != "substituters" &&
+            name != "extra-substituters")
```

Comment:
netrc-file?

Reply:
Maybe if it's relative to the flake directory, otherwise it's probably not very useful.

Reply:
I guess to make netrc-file useful we’d also need extra-sandbox-paths?

Btw. I wonder why the path of netrc-file is not automatically added to sandbox? Is there any use of setting this option without adding the path to sandbox?

Reply:
netrc-file should only be read by the Nix builtin fetchers which aren't subject to sandboxing rules.